modes.tex

```diff
@@ -222,13 +225,13 @@ \subsection{Enabling MEGA65 Extra Registers}
 \end{tcolorbox}
 
 The above example has the loop count backwards (from 255 to 0), so that your don't end up with only a
-tiny sliver of the text visible. You can make it go forwards if you like. If you do get stuck with a sliver
-of the screen, you can press RUN/STOP and RESTORE. You might be wondering: Why does RUN/STOP
-and RESTORE work when these are MEGA65 / VIC-IV registers that the C64-mode BASIC and KERNAL
+tiny sliver of the text visible. You can make it go forwards if you like. If you do get stuck with only a sliver
+of the screen, you can press \specialkey{RUN STOP} and \widekey{RESTORE}. You might be wondering: Why does \specialkey{RUN STOP}
```

Comment:
I noticed in some other places you had {RUN\\\\STOP}, would you prefer it to be the same here too?

Reply:
I'm going to replace occurrences of `RUN\\STOP` with `RUN STOP` to make the tex files a little neater.